store/timer_layer.go

```diff
@@ -2938,6 +2936,22 @@ func (s *TimerLayerGroupStore) GetMemberUsersPage(groupID string, page int, perP
 	return resultVar0, resultVar1
 }
 
+func (s *TimerLayerGroupStore) PermanentDeleteMembersByUser(userId string) *model.AppError {
```

Comment:
Is this related to the File Info store cache?

Reply:
Not really, but is generated because there is this new store API endpoint, is not necesarily something that should be here, but is harmless, anyway, I don't mind to removed it from here and add it again in another PR.